app.js

```diff
@@ -57,7 +57,7 @@ app.post('/api/v1/palettes', async (request, response) => {
 
   for (let requiredParameter of ['palette_name', 'color_one', 'color_two', 'color_three', 'color_four', 'color_five']) {
     if (!palette[requiredParameter]) {
-      return response.status(422).send({ error: `Expected format: { project_name: <string>, color_one: <string>, color_two: <string>, color_three: <string>, color_four: <string>, color_five: <string> }. You're missing a "${requiredParameter}" property.` })
+      return response.status(422).send({ error: `Expected format: { project_id: <integer> project_name: <string>, color_one: <string>, color_two: <string>, color_three: <string>, color_four: <string>, color_five: <string> }. You're missing a "${requiredParameter}" property.` })
```

Comment:
I like the changes with project_id being added to the request.body. makes total sense